backend/src/facilities/keycloak.js

```diff
@@ -81,7 +81,11 @@ const verifyUserExists = (token) => {
           if (0 === user.length) {
             userModel
               .addOne(userPayload)
-              .then((id) => resolve(`New user added to database. ID ${id}`))
+              .then(async (id) => {
+                await userModel.addRoleToOne("subscriber", id[0]);
```

Comment:
not understanding the logic here, why you are adding a subscriber, is this because this is going to be the default, and you will simply remove the admin.?

Reply:
i removed the hierarchy for now so that it is flat; that facilitates the need for adding the subscriber too. this actually has its advantages: if we put a "can_login" capability on subscriber, all we have to do to revoke someone's login, is remove the subscriber role from them. if it was hierarchical, removing subscriber would stop everyone from being able to log in.